src/form/acp.jl

```diff
@@ -114,8 +114,8 @@ function constraint_mc_slack_power_balance(pm::_PM.AbstractACPModel, nw::Int, i:
 
     end
 
-    con(pm, nw, :lam_kcl_r)[i] = cstr_p
-    con(pm, nw, :lam_kcl_i)[i] = cstr_q
+    con(pm, nw, :lam_kcl_r)[i] = isa(cstr_p, Array) ? cstr_p : [cstr_p]
+    con(pm, nw, :lam_kcl_i)[i] = isa(cstr_q, Array) ? cstr_q : [cstr_q]
```

Comment:
If the constraint is for some reason [0, 0, 0] .== [0, 0, 0], JuMP automatically makes it 0 .== 0, which makes it not an array, and these lines fail with a type error

Reply:
I just did this test,
```
using JuMP # v0.21.3
m = Model()
c = @constraint([0, 0, 0] .== [0, 0, 0])
length(c) # is 3
```
Maybe a recently fixed bug?

Reply:
Hmm, you are right, I don't get the error with that simple example, so the problem must be deeper, because I have confirmed with a different network that I still get the error with the most recent version of JuMP. I will investigate further.